src/PockyBot.NET/Services/UserLocationGetter.cs

```diff
@@ -0,0 +1,91 @@
+using System.Text;
+using PockyBot.NET.Constants;
+using PockyBot.NET.Persistence.Repositories;
+
+namespace PockyBot.NET.Services
+{
+    public class UserLocationGetter : IUserLocationGetter
+    {
+        private readonly IPockyUserRepository _pockyUserRepository;
+
+        internal UserLocationGetter(IPockyUserRepository pockyUserRepository)
+        {
+            _pockyUserRepository = pockyUserRepository;
+        }
+
+        public string GetUserLocation(string[] commands, string[] mentionedUsers, string meId)
+        {
+            if (commands.Length < 1)
```

Comment:
what if there's more than one command?

Reply:
I've designed this to be as forgiving as possible. It takes the first command and discards the rest. Wanted it to be more chatbot-like than like calling a strict function.